pkg/operator/ceph/file/mds/mds.go

```diff
@@ -141,6 +141,15 @@ func (c *Cluster) Start() error {
 			return fmt.Errorf("failed to generate keyring for %q. %+v", resourceName, err)
 		}
 
+		// Set ceph mds config flag before it starts
+		// This works since the keyring has been created
+		if c.clusterInfo.CephVersion.IsAtLeastMimic() {
+			err = c.setDefaultFlagsMonConfigStore(mdsConfig.DaemonID)
```

Comment:
This will run on every orchestration. Won't that overwrite the value if the user sets it in the mon store? Is the intent only to run this once?

Reply:
Yes, this will run at every orchestration, let me see how can I make it a one-shot.